### PR TITLE
Merge 3.x

### PR DESCRIPTION
**Pull request by greg0ire**
Maintenance merge

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,29 +37,29 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.3'
```
Comment (jordisala1991) [inline .travis.yml]: This should be 7.0

File: .travis.yml [hunk 2]
```diff
@@ -15,6 +15,7 @@ php:
   - '5.5'
   - '5.6'
```
Comment (jordisala1991) [inline .travis.yml]: This and the above should be droped

File: .travis.yml [hunk 3]
```diff
@@ -36,29 +37,29 @@ env:
...
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.3'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
+    - php: '7.1'
       env: SYMFONY=2.3.*
-    - php: '7.0'
+    - php: '7.1'
       env: SYMFONY=2.7.*
```
Comment (jordisala1991) [inline .travis.yml]: This version should be droped

**Comment by greg0ire** [comment]
RTM @sonata-project/contributors ?

**Comment by jordisala1991** [comment]
Thanks @greg0ire

**Pull request merged**